### DISCUSSION TITLE
Crossfading

### DISCUSSION THREAD

**Discussion by sashahilton00** [librespot-org/librespot#643] [Feature Requests]
<a href="https://github.com/niklasdoerfler"><img src="https://avatars2.githubusercontent.com/u/173291?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [niklasdoerfler](https://github.com/niklasdoerfler)**
_Sunday Jun 26, 2016 at 10:53 GMT_
_Originally opened as https://github.com/plietar/librespot/issues/87_

----

Hey,

it would be nice, if you could implement the ability to set up some fading between two tracks (as the original client does). Is that possible?

Thanks!
Niklas

**Comment by sashahilton00** [comment]
<a href="https://github.com/plietar"><img src="https://avatars0.githubusercontent.com/u/1489775?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [plietar](https://github.com/plietar)**
_Wednesday Jul 06, 2016 at 01:59 GMT_

----

Related #18

**Comment by benjamin-kirkbride** [comment]
Any update to this? Would be great to have

**Comment by AxelPuntke** [comment]
Would be a very great feature, are there some plans to implement it in the future?

**Reply by roderickvd** [reply]
To be absolutely honest it is not something that scratches my itch, and so not something on my list also because it is not totally trivial to do. But if anyone wants to work on it, I would very much welcome PRs.

**Reply by kingosticks** [reply]
If there were PRs for this, should they be on the new api branch or current dev? (To be clear, I'm not going to work on this)

**Reply by roderickvd** [reply]
I would prefer `new-api` for anything non-trivial, which I think it would be.
